ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -119,10 +120,43 @@ then
 	then
 		if fmu mode_pwm
 		then
-			mixer load /dev/pwm_output1 $MIXER_AUX_FILE
+			mixer load $OUTPUT_AUX_DEV $MIXER_AUX_FILE
 		else
 			tone_alarm $TUNE_ERR
 		fi
+
+		if [ $PWM_AUX_OUT != none ]
```

Comment:
Added them in the same commit https://github.com/anton-matosov/Firmware/commit/2b13cd3706fb1ffe569c1dbc3e7a42425b4be6bc